config/locales/en.yml

```diff
@@ -40,7 +40,15 @@ en:
         by <a href='http://www.michaelhartl.com/'>Michael Hartl</a>"
       news: "News"
     user:
-      title: "Sign up"
-      welcome_text: "This will be a signup page for new users."
+      signup:
+        title: "Sign up"
+        button_regis: "Create my account"
+        form_field_name: "Name"
+        form_field_email: "Email"
+        form_field_password: "Password"
+        form_field_confirmation: "Confirmation"
+        message_success: "Create account success, Welcome to the Sample App!"
+        error_messages: "The form contains "
```

Comment:
truyền biến count vào rồi in ra trên 1 message chứ ko tách rời như hiện tại nha e. Tìm hiểu cách thiết lập số ít số nhiều  trong i18n